dev-libs/gobject-introspection/gobject_introspection-1.55.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Introspection system for GObject-based libraries"
+DESCRIPTION="GObject introspection is a middleware layer between C libraries \
+(using GObject) and language bindings. The C library can be scanned at \
+compile time and generate a metadata file, in addition to the actual native \
+C library. Then at runtime, language bindings can read this metadata and \
+automatically provide bindings to call into the C library."
+HOMEPAGE="https://wiki.gnome.org/Projects/GObjectIntrospection"
+COPYRIGHT="Colin Walters
+	Johan Dahlin
+	Matthias Clasen (girepository)
+	Jürg Billeter (scanner)"
+LICENSE="GNU LGPL v2
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/1.55/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="888fba5df3ebfa42ea9f24ad98a667fdc151cf37a07349352804028a5ef0948f"
+SOURCE_DIR="gobject-introspection-$portVersion"
+PATCHES="gobject_introspection-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gobject_introspection$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:g_ir_annotation_tool$secondaryArchSuffix
+	cmd:g_ir_compiler$secondaryArchSuffix
+	cmd:g_ir_generate$secondaryArchSuffix
+	cmd:g_ir_inspect$secondaryArchSuffix
+	cmd:g_ir_scanner$secondaryArchSuffix
+	lib:libgirepository_1.0$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gobject_introspection${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libgirepository_1.0$secondaryArchSuffix
+	devel:_giscanner$secondaryArchSuffix
+	"
+
+REQUIRES_devel="
+	gobject_introspection$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcairo$secondaryArchSuffix
+	devel:libffi$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:bison
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:aclocal
+	cmd:autoconf
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
+	cmd:which
+	"
+
+BUILD()
+{
+	#export CAIRO_LIBS="-lcairo -lcairo-gobject"
+	#export CAIRO_CFLAGS="-I$developDir/cairo"
+
+	runConfigure ./autogen.sh CPPFLAGS=-I$developDir/python3.6m/ \
+	PYTHON=python3.6m --with-cairo=yes
+	make
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool library files
+	rm $libDir/lib*.la
```

Comment:
The only non neutral change is the one for the `rm`.
```diff
-	rm $libDir/lib*.la
+	rm -f "$libDir"/lib*.la "$libDir"/gobject-introspection/giscanner/*.la
 
-	prepareInstalledDevelLibs libgirepository-1.0
+	prepareInstalledDevelLib libgirepository-1.0
-
 	fixPkgconfig
```
(The other changes are just suggestions.)